main/postgresql/APKBUILD

```diff
@@ -3,13 +3,13 @@
 # Contributor: Jakub Jirutka <jakub@jirutka.cz>
 pkgname=postgresql
 pkgver=11.4
-pkgrel=0
+pkgrel=1
 pkgdesc="A sophisticated object-relational DBMS"
 url="https://www.postgresql.org/"
 arch="all"
 license="PostgreSQL"
 depends="postgresql-client tzdata"
-install="$pkgname.pre-upgrade"
+install="$pkgname.pre-install $pkgname.pre-upgrade"
 pkgusers="postgres"
```

Comment:
I always get confused about `pkgusers`/`pkggroups` is that still needed when creating the users in a `pre-install` script?

Reply:
The `pre-install` script will not run when you build the package, so if the `postgres` user does not exist on the builder, things like `chown postgres` in APKBUILD will fail. abuild will create the `pkgusers` on the building system. `pre-install` creates it on the runtime system.